### DISCUSSION TITLE
supporting hex colors

### DISCUSSION THREAD

**Discussion by Redhawk18** [Ogeon/palette#358] [Ideas]
I wanted to make a discussion since this issue is old #306 

I understand how you feel about not wanting to support a billion different ways to compose a string, but support hex colors seems important at least.

[Currently I'm using this crate](https://docs.rs/cursive/0.20.0/cursive/theme/enum.Color.html#method.parse) and if you want to support all types of strings maybe this could be a good idea.

**Comment by Ogeon** [comment]
Hi! Don't worry about commenting on an older issue (it's also only from March). :slightly_smiling_face: I try to only keep relevant issues around.

As I commented in https://github.com/Ogeon/palette/issues/306#issuecomment-1732367442, hex parsing is already implemented, but unfortunately not very visible unless you look for the `FromStr` trait implementation. I'm all for supporting formats separately (provided they aren't too complex, which the CSS syntax might be) and I think what probably makes it hard to find is probably that it's buried in the traits section under a name that isn't obvious. I hope my example there helped.
